### ISSUE TITLE
Announcement

### ISSUE THREAD

**Issue by cuekYT** [cuekYT/GTproxy#11]
There are mentally disabled slaves of a disabled retard who claims that my work is his and it is not mine. I spent hours/days/weeks on my projects (GTinternal & GTproxy) and he is claiming that it is his. Please, ignore them.

**Comment by ymeedo** [comment]
I have a few questions for you.

- Why does GitHub say you joined 12 hours ago?
- Why was this repository just made and has very few commits, yet it's at version 2?
- Why do you call people "mentally disabled slaves of a disabled retard"? You should be more considerate to people with mental disabilities.
- @playingoDEERUX's repository actually has way more stars and a deeper history of commits, yet, your source code and his looks exactly the same, which I consider proof enough that this indeed @playingoDEERUX. Why are you denying this?
- I've seen a few issues from iProgramMC commenting here, and they have all been deleted. I've also seen in @playingoDEERUX's issues, that his issue was edited to "im slave" "i suck playingos dick" which is funny to me, because you seem to be very immature.

TL;DR: Your allegations that @playingoDEERUX stole your code are proven false, this, you can expect a DMCA takedown notice in the next 168 hours.
